src/styles/marker.styles.ts

```diff
@@ -0,0 +1,7 @@
+export const ARROW_MARKER: Object = {
+  backgroundColor: "transparent",
+  borderWidth: 4,
+  borderColor: "transparent",
+  borderTopColor: "#AA2B45",
```

Comment:
Not a big dead since arrow_marker is already a constant, but this color string could be moved to a constant.

Reply:
Will leave it as is since its more tied to the marker and won't be re-used anywhere else